xmtp_id/src/associations/unsigned_actions.rs

```diff
@@ -104,63 +121,67 @@ impl SignatureTextCreator for UnsignedIdentityUpdate {
             .map(|action| action.signature_text())
             .collect::<Vec<String>>();
         format!(
-            "I authorize the following actions on XMTP:\n\n{}\n\nAuthorized at: {}",
+            "{HEADER}\n\nInbox ID: {}\nCurrent time: {}\n\n{}\n\n{FOOTER}",
+            self.inbox_id,
+            ns_date_to_iso_8601(self.client_timestamp_ns),
             all_signatures.join("\n\n"),
-            // TODO: Pretty up date
-            self.client_timestamp_ns
         )
     }
 }
 
+fn ns_date_to_iso_8601(ns_date: u64) -> String {
+    let date = DateTime::from_timestamp_nanos(ns_date as i64);
+    date.to_string()
```

Comment:
This isn't actually ISO8601, it looks like this: `2015-05-15 00:00:00 UTC`

I actually like it more though because it's way more human readable, I can update the XIP to match. But we should rename this method in that case

Reply:
Also, would it make sense to round the nanoseconds to seconds, before feeding it into `DateTime`? Looks like it's sometimes rendered with decimal places, e.g. `2000-01-12 01:02:03.004 UTC`, which might cause some clients to fail validation

Reply:
OK. I've fixed the ISO formatting and added rounding to seconds.